indico/modules/events/contributions/util.py

```diff
@@ -71,3 +72,11 @@ def serialize_contribution_person_link(person_link):
     data['authorType'] = person_link.author_type.value
     data['isSubmitter'] = person_link.is_submitter
     return data
+
+
+def serialize_subcontribution_person_link(person_link):
```

Comment:
No need for this, since we are keeping submission rights for SubcontributionPersonLink.